### ISSUE TITLE
Class property `param type annotation required` not working

### ISSUE THREAD

**Issue by amilajack** [gajus/eslint-plugin-flowtype#89]
The following code will not throw an error with the given rule:

`"flowtype/require-parameter-type": 2,`

```js
class Example {
  number = 3 // <-- does not throw error
}
```

**Comment by amilajack** [comment]
Never mind. I realized now that this rule only lints parameter types.
